tests/test_validators.re

```diff
@@ -90,6 +96,12 @@ describe("validators", ({test, _}) => {
 
     // ensure this is a noop
     expect.equal(update_current(b.address, t), t);
+
```

Comment:
Can we have more descriptive test descriptions, atleast for the touched tests?

Reply:
For instance, this test could read "Only validators present in the authorised list can be made current validator". This could be documented in the doc string at the `update_current` function too.

Reply:
openen an issue as a follow up #56